lrpar/src/lib/ctbuilder.rs

```diff
@@ -802,8 +863,9 @@ where
             };
             outs.push_str(&format!(
                 "    // {rulename}
-    #[allow(clippy::too_many_arguments)]
-    fn {prefix}action_{}<'lexer, 'input: 'lexer>({prefix}ridx: ::cfgrammar::RIdx<{storaget}>,
+    #[allow(clippy::too_many_arguments, unused_variables)]
+    fn {prefix}action_{}<'lexer, 'input: 'lexer, {param_lt}>(
```

Comment:
Why allow `unused_variables`?

Reply:
I had tried to not do this but hadn't figured out a way, this is because the param is being passed through to user action code, and so user actions like `{ $1 }` will trigger this lint.

I think one way to disable this in user-action code is force the user-code to `{ #[allow(unused_variables) { $1 } }`, but that didn't occur to me until just now.